java/src/main/resources/template.properties

```diff
@@ -164,6 +164,15 @@ gcs.jdbc.output.url=<output jdbc url>
 gcs.jdbc.output.table=<jdbc table id>
 gcs.jdbc.output.saveMode=<Append|Overwrite|ErrorIfExists|Ignore>
 
+# GCS to GCS
+gcs.gcs.input.location=<gcs path for input file>
```

Comment:
please only specify default values in the property file and keep it blank for required properties
Follow the example of "JDBCToBQ Template properties" in this file

Reply:
Done.